Gordon360/Controllers/RecIM/TeamsController.cs

```diff
@@ -186,5 +186,32 @@ public ActionResult<IEnumerable<TeamInviteViewModel>> GetTeamInvites()
             var username = AuthUtils.GetUsername(User);
             return Ok(_teamService.GetTeamInvites(username));
         }
+
+        [HttpPatch]
+        [Route("{teamID}/accepted")]
+        public async Task<ActionResult<TeamInviteViewModel>> AcceptTeamInvite(int teamID, [FromBody] ParticipantTeamUploadViewModel acceptedInvite)
+        {
+            var username = AuthUtils.GetUsername(User);
+            if (acceptedInvite.Username != username)
+            {
+                return Forbid($"You are not permitted to accept invitations for {acceptedInvite.Username}.");
+            }
+
+            // set the role type ID of the accepted team invite to 3 => member
+            acceptedInvite.RoleTypeID = 3;
+            var joinedParticipantTeam = await _teamService.UpdateParticipantRoleAsync(teamID, acceptedInvite);
+
+            // true delete other team invites from the same activity
+            IEnumerable<TeamInviteViewModel> teamInvites = _teamService.GetTeamInvites(username);
```

Comment:
You probably only want to delete the team invite objects for the specific sport/league, right? What if a person is playing in two different recim sports at the same time? (I'm not 100% sure if Gordon Rec/IM allows that or not, but either way that case shouldn't be handled here)

Reply:
You are right. We should only true delete the team invite from the same activity.